test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -8575,7 +8575,7 @@ def cleanup_jobs(self):
         jobs. Specifying an extend parameter could override
         this behavior.
         """
-        delete_xt = 'nomailforce'
+        delete_xt = 'force'
```

Comment:
Hi @agrawalravi90 This was added as part of supporting suppress_mail option while deleting Job in framework. But according to our document, we are not supposed to use -Wsuppress_email=-1 option with -Wforce. This qdel command was passing before because there was a bug in -Wsuppress_email=-1 which got fixed recently and hence, our qdel command in tearDown/setUp started failing.